src/Symfony/Component/HttpClient/Response/CurlResponse.php

```diff
@@ -221,6 +220,7 @@ public function getInfo(?string $type = null): mixed
                 rewind($this->debugBuffer);
                 ftruncate($this->debugBuffer, 0);
             }
+            $this->info = array_merge($this->info, $info);
```

Comment:
Before #60241 `getInfo()` could have been called multiple times but the debug info would always be present as the debug buffer was never truncated and thus preserved for subsequent calls. This is no longer the case as we truncate the buffer two lines above just after reading it. Following calls of `getInfo()` would this no longer read the debug information from the buffer but we need to store it at some other place.